### PR TITLE
Implements new Dominia Sprites

### PR DESCRIPTION
**Pull request by Geevies**
On mobile at the moment, screenshots will come soon.

### CODE REVIEW

File: html/changelogs/geeves-dommy.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Geeves
```
Comment (Geevies) [inline html/changelogs/geeves-dommy.yml]:
```suggestion
author: Code by Geeves, Sprites by Cre77
```

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
Dominia Clothes
image

Lyodsuits
image

**Comment by Geevies** [comment]
Thread: https://forums.aurorastation.org/topic/13803-replace-the-current-dominian-great-coat-and-red-suit-sprite-with-these-here-posted

File: code/modules/clothing/factions/dominia.dm [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+/obj/item/clothing/under/dominia/black
+	icon = 'icons/clothing/under/uniforms/dominia_uniform_black.dmi'
+	icon_state = "dominia_uniform_black"
+	item_state = "dominia_uniform_black"
+
+/obj/item/clothing/under/dominia/sweater
+	name = "dominia sweater"
+	desc = "This is a sweater of Dominian style."
+	icon = 'icons/clothing/under/uniforms/dominia_sweater.dmi'
+	icon_state = "dominia_sweater"
```
Comment (fernerr) [inline code/modules/clothing/factions/dominia.dm]:
```suggestion
	icon_state = "dom_sweater"
```

File: code/modules/clothing/factions/dominia.dm [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+	item_state = "dominia_sweater"
```
Comment (fernerr) [inline code/modules/clothing/factions/dominia.dm]:
```suggestion
	item_state = "dom_sweater"
```

**Pull request merged**